crypto/engine/eng_list.c

```diff
@@ -18,7 +18,7 @@
 /*
  * The linked-list of pointers to engine types. engine_list_head incorporates
  * an implicit structural reference but engine_list_tail does not - the
- * latter is a computational niceity and only points to something that is
+ * latter is a computational nicety and only points to something that is
```

Comment:
I believe that the author meant "nicety", but I think "optimization" is probably a clearer word choice.